v7/_posts/odata features/2016-09-02-06-13-support-untyped-json.md

```diff
@@ -1,15 +1,25 @@
 ---
 layout: post
-title: "Support undeclared & untyped property value in ODataLib"
+title: "Read and Write Untyped values in ODataLib"
 description: ""
 category: "5. OData Features"
 ---
 
-From ODataLib 7.0, `ODataMessageReader` & `ODataMessageWriter` are able to read & write arbitrary JSON as raw string in the payload. The undeclared & untyped property is supported by ODataLib in a slightly different way than that in ODataLib 5.x.
+In ODataLib 7.x, `ODataMessageReader` & `ODataMessageWriter` are able to read & write untyped primitive, structured, and collection values.  
```

Comment:
It's better to mention from which version we support ....

and add a link to the nuget or nightly